packages/orchestration/src/examples/stake-bld.contract.js

```diff
@@ -112,7 +113,8 @@ export const start = async (zcf, privateArgs, baggage) => {
             trace('makeStakeBldInvitation', give);
             const { holder } = await makeLocalAccountKit();
             /** @type {Record<string, Payment<'nat'>>} */
-            // @ts-expect-error XXX PaymentPKeywordRecord throught deeplyFulfilled will be a PaymnentKeywordRecord
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- fails only through some build paths
+            // @ts-ignore XXX PaymentPKeywordRecord throught deeplyFulfilled will be a PaymnentKeywordRecord
```

Comment:
Let's fix the typos. This is really hard to read as is.

```suggestion
            // @ts-ignore XXX PaymentPKeywordRecord, though after deeplyFulfilled it will be a PaymentKeywordRecord
```

Reply:
if I have to do another push I'll include this

re: `b1-${string}` above, I tried that and a bunch of places have full wide `string` or use a different literal like `id` (in a test). So out of scope

Reply:
Where do these numbers come from? What do they get compared to?

Reply:
try `git grep typeCoverage`